english/music/add_music.md

```diff
@@ -0,0 +1,11 @@
+Add Music for your cake recipe?
```

Comment:
I do not know how to do this - I'll just skip it for now - perhaps edit an existing file and add these lines, since I'm a web novice - still learning.  Thanks.